src/Controller/ProductApiController.php

```diff
@@ -110,6 +111,7 @@ public function edit(Request $request, Product $product, ProductRepository $prod
         $product->setImgPath($parameters['imgPath']);
         $product->setDescription($parameters['description']);
         $product->setUpdatedAt($dateTime);
+        $product->setAvailableAmount($parameters['availableAmount']);
```

Comment:
I think you have forgotten verification that availableAmount is integer and >=0. Or is the field avaulable amount will be required false you can add default value in product